mir_eval/separation.py

```diff
@@ -108,7 +108,7 @@ def validate(reference_sources, estimated_sources):
 
 
 def bss_eval_sources(
-    reference_sources, estimated_sources, compute_permutation=True
+        reference_sources, estimated_sources, compute_permutation=True
```

Comment:
Still bad indent style :grin: 

should be

```ptython
def bss_eval_sources(reference_sources, estimated_sources,
                     compute_permutation=True, ...
```

but i won't be a stickler on this.